tests/ecosystem/deployment/test_deployment.py

```diff
@@ -39,3 +45,40 @@ def test_deployment(pvc_factory, pod_factory):
         log.info(
             "Cluster will be destroyed during teardown part of this test."
         )
+
+
+@skipif_no_encrypted
+@deployment
+def test_verify_osd_encryption():
```

Comment:
This is problematic to have it defined as separate test case I think..

1) using skipif we will see it in the list of skipped tests for deployment all the time when we don't run against environment encrypted at rest
2) I am afraid about the teardown, that this will run as well as we run teardown like:
```
run-ci -m deployment \
    --cluster-name kerberos_ID-ocs-deployment \
    --cluster-path /home/my_user/my-ocs-dir tests/ \
    --teardown
```

Which mean's that test will run after the environment will be destroyed and will fail IMO.

I would add this check directly to the block after line 42:
https://github.com/red-hat-storage/ocs-ci/pull/3142/files#diff-572009fd6960f0b3ddfbf7a115af85876bdd2331d90084d188f96d59e604c98bR42

So you can keep it as separate function, and call it from there, but I would avoid have it as separate test case cause of the issues mentioned above.

Reply:
As we discussed offline, we decided to move it to ocs_install_verification , just after FIPS verification. And not have it as separate test case.

Update about teardown - it will probably not fail the destroy as cluster is session scope autouse fixture, but it will run this verification test also in destroy which is not intention here. So with the move to ocs_install_verification we solve everything.

Thanks